x-pack/plugins/security_solution/public/timelines/containers/active_timeline_context.ts

```diff
@@ -17,9 +17,15 @@ import { TimelineEventsAllRequestOptions } from '../../../common/search_strategy
  * I did not want to put in the store because I was feeling it will feel less temporarily and I did not want other engineer using it
  *
  */
+
+export interface ActiveTimelineExpandedEvent {
```

Comment:
related to this one https://github.com/elastic/kibana/pull/82033/files#r524533002